public/app/features/storage/storage.ts

```diff
@@ -131,6 +131,10 @@ class SimpleStorage implements GrafanaStorage {
       path += '.json';
     }
 
+    if (!path.startsWith('content/')) {
```

Comment:
:(

we still need to upload files to "resources" that will not live under content

Reply:
nevermind!